__tests__/app.test.js

```diff
@@ -109,6 +108,51 @@ describe("/articles/:article_id", () => {
   });
 });
 
+describe("/articles/:article_id/comments", () => {
+  test("GET:200 sends an array of comments for the given article_id", () => {
+    return req(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.comments).toEqual([
+          {
```

Comment:
We could check using a forEach to make the test more robust in case any changes are made in the future 🙂